server/src/servershot.js

```diff
@@ -126,6 +141,15 @@ class Shot extends AbstractShot {
           if (! rowCount) {
             throw new Error("No row updated");
           }
+          return db.queryWithClient(
+            client,
+            "UPDATE content SET head = $1, body = $2 WHERE shotid = $3",
+            [head, body, this.id]
+          );
+        }).then((rowCount) => {
+          if (! rowCount) {
+            throw new Error("No row updated");
```

Comment:
Some failure could cause this (e.g., something that causes the first insert to work without the second insert work).  It would be better to warn and insert.  But it also wouldn't really be a problem if it was one table.